Library/Formula/zorba.rb

```diff
@@ -0,0 +1,46 @@
+
+class Zorba < Formula
+  homepage "http://www.zorba.io/"
+  url "https://github.com/28msec/zorba/archive/3.0.tar.gz"
+  sha1 "c444cde689600aab1172b4974348dd6626e92261"
+
+  option "disable-big-integer", "Use 64 bit precision instead of arbitrary precision for performance"
+  option "enable-ssl-verification", "Enable SSL peer certificate verification"
+
+  depends_on "cmake" => :build
+  depends_on "swig" => :build
+  depends_on "curl"
+  depends_on "libxml2"
+  depends_on "flex"
+  depends_on "icu4c"
+  depends_on "xerces-c"
+
+  # While it depends on Bison, Zorba 3.0 is not compatible with Bison 3.0.
+  # We will therefore default to the version bundled with Zorba (Bison 2.3).
+  # <https://bugs.launchpad.net/zorba/+bug/1227047>
+
+  def install
+
+    ENV["CMAKE_PREFIX_PATH"] = "#{Formula["icu4c"].prefix}:#{Formula["flex"].prefix}"
+
+    args = std_cmake_parameters.split
```

Comment:
This should probably be ` std_cmake_args `, which should cover the ` Release ` element as well.